aeronet_raster/aeronet_raster/utils/calc_window_weight_mtrx.py

```diff
@@ -0,0 +1,177 @@
+import numpy as np
+
+
+def calc_weight_item(x, sample_size_x, bound):
+    """
+        Calculate weight depending on x coordinate
+    """
+    if x < 2 * bound:
+        w_x = x / (2 * bound)
+    elif 2 * bound <= x < sample_size_x:
+        w_x = 1
+    else:
+        w_x = (sample_size_x + 2 * bound - x) / (2 * bound)
+
+    return w_x
+
+
+def calc_weight_mtrx(sample_size, bound):
```

Comment:
Looks like python for-loops can be not optimal here, can we create values via numpy, like using numpy.linspace etc?

However, if it is done once on creation of Writer, it may be not important. Did you profile execution time?

Reply:
I think  it is optimal now, but i added # todo